docs/docs/contributing.md

```diff
@@ -0,0 +1,73 @@
+---
+slug: /contributing/
+sidebar_position: 7
+---
+
+# Contributing
+
+Contributions to Task are very welcome, but we ask that you read this document before submitting a PR.
+
+## Before you start
+
+- **Check existing work** - Is there an existing PR? Are there issues discussing the feature/change you want to make? Please make sure you consider/address these discussions in your work.
+- **Backwards compatibility** - Will your change break existing Taskfiles? It is much more likely that your change will merged if it backwards compatible. Is there an approach you can take that maintains this compatibility? If not, consider opening an issue first so that API changes can be discussed before you invest you time into a PR.
+
+## 1. Setup
+
+- **Golang** - Task is written in [Go]. You will need to [install version 1.18+].
+- **Node.js** - [Node.js] is used to host Task's documentation server and is required if you want to run this server locally.
+- **Yarn** - [Yarn] is the Node.js package manager used by Task.
+
+## 2. Making changes
+
+- **Code style** - Try to maintain the existing code style where possible and ensure that code is formatted by `gofmt`.
+- **Documentation** - Ensure that you add/update any relevant documentation. See the [updating documentation](#updating-documentation) section below.
+- **Tests** - Ensure that you add/update any relevant tests and that all tests are passing before submitting the PR. See the [writing tests](#writing-tests) section below.
+
+### Running my changes
+
+To run Task with working changes, you can use `go run ./cmd/task`. To run a development build of task against a test Taskfile in `testdata`, you can use `go run ./cmd/task --dir ./testdata/<my_test_dir> <task_name>`.
+
+### Updating documentation
```

Comment:
A convention that needs to be mentioned here is trying to wrap most lines in 80 chars, a common convention for markdown files.

Also, can you do that for this file as well? Thanks 😄